### DISCUSSION TITLE
Redash loading results problem

### DISCUSSION THREAD

**Discussion by fmaulanaaaa** [getredash/redash#6206] [Q&A]
Hello everyone, 
i have a problem on redash, when i'm using query to bigquery it will return **Error running query: failed communicating with server. Please check your Internet connection and try again.**. 
It only happen when i'm trying to query with 2 months++, like january until march. 

is there a way to troubleshoot this? i have set the REDASH_LOG_LEVEL to ERROR, but still no clue

**Comment by gaecoli** [comment]
How do you install redash? And what is your env?

**Comment by fmaulanaaaa** [comment]
i'm installing redash on GKE using helm chart https://getredash.github.io/contrib-helm-chart/ version 3.0.0

it's production env
